### PR TITLE
ci: Test MacOS build

### PR DESCRIPTION
**Pull request by juan-lunarg**

### CODE REVIEW

**Comment by juan-lunarg** [comment]
Tested in my fork:
https://github.com/juan-lunarg/Vulkan-Profiles/actions/runs/4226455029

File: CMakeLists.txt [hunk 1]
```diff
@@ -48,7 +46,7 @@ if (UPDATE_DEPS)
     message("********************************************************************************")
 
     set(optional_args)
-    if (NOT BUILD_TESTS)
```
Comment (christophe-lunarg) [inline CMakeLists.txt:51]: Is this actually a good things this variable is not name BUILT_TESTS? Isn't it inconsistent with other repository?

**Comment by christophe-lunarg** [comment]
If we could have macOS C.I., that would be nice!

File: CMakeLists.txt (hunk 1 above)
Comment (juan-lunarg) [reply CMakeLists.txt:51]: It is inconsistent. But in this case I'd argue it's a good thing. It's generally good practice to group your cache variables together so it shows up nicely in cmake-gui.

image

Comment (juan-lunarg) [reply CMakeLists.txt:51]: Main reason I haven't fixed it in vvl is because it's already established.

**Pull request merged**